src/lib/layers/lane_details/LanePolygons.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import Layer from "./Layer.svelte";
+  import { caseHelper } from "../../../maplibre_helpers";
+  import type { GeoJSON } from "geojson";
+
+  export let gj: GeoJSON;
+
+  let style = {
+    type: "fill",
+    paint: {
+      "fill-color": caseHelper(
+        "type",
+        // TODO Generate TS types from osm2streets
+        {
+          Driving: "black",
+          Parking: "#333333",
+          Sidewalk: "#CCCCCC",
+          Shoulder: "#CCCCCC",
+          Biking: "#0F7D4B",
```

Comment:
This is an old US assumption! Is there a standard color for cycle lanes in the UK? I feel like I've seen both blue and red and just markings in London alone...

Reply:
Page 70 of [LTN1/20](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/951074/cycle-infrastructure-design-ltn-1-20.pdf) doesn't obligate a colour choice in the UK, but does mention green and red surfaces. 

My vote on this would be green!